collector/compile-benchmarks/tt-muncher/REUSE.toml

```diff
@@ -0,0 +1,18 @@
+version = 1
```

Comment:
I think it's fine to inline this into the parent `REUSE.toml`.

Reply:
Done! Thanks.